bokehjs/src/coffee/common/plot.coffee

```diff
@@ -122,6 +123,11 @@ class PlotView extends ContinuumView
 
     @update_dataranges()
 
+    if @mget('responsive')
+      throttled_resize = _.throttle(@resize, 100)
+      $(window).on("resize", throttled_resize)
+      $(@resize)
```

Comment:
Absolutely. See note at start of PR. Can't wait for this to do more. If someone was feeling sneaky in the short-run and had the ability to know when their container resized they could always to a `$(window).trigger('resize')` :D